bank/src/main/java/com/clouway/bricky/core/db/balance/BalanceRepository.java

```diff
@@ -0,0 +1,15 @@
+package com.clouway.bricky.core.db.balance;
+
+import com.clouway.bricky.core.user.CurrentUser;
+
+/**
+ * @author Marian Zlatev <mzlatev91@gmail.com>
+ */
+public interface BalanceRepository {
+
+  CurrentUser depositToCurrentUser(double amount);
+
+  CurrentUser withdrawFromCurrentUser(double amount) throws FundDeficitException;
+
+  CurrentUser getCurrentUser();
+}
```

Comment:
No I can't perform operations on user other than CurrentUser. Currently my back is only working with the loged in user. If I have to do transactions from one account to another maybe i will have a TransactionRepository or something like that that will perform transactions from one used account to another.